js/samples/04.ai-apps/e.assistants-orderBot/teamsapp.local.yml

```diff
@@ -68,11 +68,13 @@ provision:
 # Defines what the `deploy` lifecycle step does with Teams Toolkit.
 # Runs after `provision` during Start Debugging (F5) or run manually using `teamsfx deploy --env local`.
 deploy:
+    # Set `--workspaces=true` to use local bits
```

Comment:
This is incorrect. Please delete this line.

Reply:
fixed